### DISCUSSION TITLE
Account suspended

### DISCUSSION THREAD

**Discussion by ghost** [community/community#130918] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hi,

My GitHub account: https://github.com/hv33y was suspended. 
My ticket: `2867879`
My Email: `github@harrykauhaad.com`

I don't know why as I didn't receive any mails or hints indicating why this happened.
I was taking to some one here in the community trying to solve a problem when suddenly it logged me out and when tried to sign in a message appear that account suspended for violating GitHub’s terms of service. Can some one please what is happening?
I am so sad and can't describe my sadness. I never violated TOS in my life and if that happened it is absolutely a mistake and not intentional. I’m happy to oblige with all of GitHub’s policies and Terms.

Thank you!

@github

**Comment by asdacosta** [comment]
It happens often and unfortunately even if you submit a request to know why your account was suspended, you’ll receive no feedback. 
GitHub keeps updating their terms and policies in the community here, so you can check to see if you mistakenly violated any new one. 
Most accounts are actually suspended because they use chat bots to answer questions they know nothing about.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @hkauhaad, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

_Please note for future posts: You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them._
